<launch>
  <node name="DualCam" pkg="mc_vslam" type="DualCam" args = "config_gaz_exp.yaml" output="screen" >
  </node>
</launch>